gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3405,6 +3431,18 @@ subroutine Run ( gc, import, export, clock, rc )
                list(n)%unit = -1
             end if
             list(n)%currentFile = filename(n)
+         elseif (list(n)%observation_spec /= '') then
+            if (list(n)%observation_spec == 'station') then
+               if (list(n)%unit.eq.0) then
+                  if (mapl_am_i_root()) write(6,*) "Station_data output to new file: ",trim(filename(n))
```

Comment:
Use pflogger here.

Reply:
Use pflogger here instead.


Also, open another ticket to deal with any legacy WRITE statements.    (Probably assigned to someone else.)

Reply:
done